components/automate-cli/cmd/chef-automate/verify.go

```diff
@@ -241,7 +240,14 @@ func (v *verifyCmdFlow) RunVerify(config string) error {
 	}
 
 	// Get config required for batch-check API call
-	batchCheckConfig := &models.Config{}
+	batchCheckConfig := &models.Config{
```

Comment:
Why are we adding components here?

Reply:
@YashviJain01 since nested struct are passed by reference if if we don't initialise the nested  struct then it will throw SEGMENTATION NIL POINTER ERROR! and unit test cases will fail. batchCheckConfig := &models.Config is coming from batch check and to make the test cases pass in pipeline I had to correct it.